src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -312,34 +308,37 @@ public string PutCounter(DocumentsOperationContext context, string documentId, s
             }
         }
 
-        private static int GetDbIdIndex(DocumentsOperationContext context, BlittableJsonReaderObject data)
+        private int GetDbIdIndex(BlittableJsonReaderObject data)
         {
             var dbIdIndex = int.MaxValue;
             if (data.TryGet(DbIds, out BlittableJsonReaderArray dbIds))
             {
                 for (dbIdIndex = 0; dbIdIndex < dbIds.Length; dbIdIndex++)
                 {
-                    if (dbIds[dbIdIndex].Equals(context.Environment.Base64Id) == false)
+                    if (dbIds[dbIdIndex].Equals(_localNodeTagAndDbId) == false)
```

Comment:
We should hold only the db id here, not the node tag

Reply:
done